kvbc/include/block_digest.h

```diff
@@ -9,8 +9,6 @@
 
 namespace concord::kvbc {
```

Comment:
Do we need this file just for single declaration?
I think it can be moved to some other header.

Reply:
I suggest to keep it as is because it is used only within `kvbc` folder.